src/app/debug/table/table.component.ts

```diff
@@ -446,4 +455,37 @@ export class TableComponent implements OnInit, OnDestroy {
   getCheckBoxSize() {
     return `${13 + this.tableSpacing}px`;
   }
+
+  setUniqueOptions(data: any): void {
```

Comment:
Why is data any?

Reply:
because there's no interface or class for the data, just like everywhere else in the project